--- login.php.orig	2005-06-22 16:45:35.000000000 +0300
+++ login.php	2005-08-04 19:52:57.089477776 +0300
@@ -146,7 +146,8 @@
                     $rcptaddress . 
 				    addHidden('just_logged_in', '1'),
                                 'left', '', 'width="*"' )
-                            ) ,
+                            ) .
+			    concat_hook_function('login_form'),
                         'center', '#ffffff', 'border="0" width="100%"' ) ,
                     'left', '#FFFFFF' )
                 ) . 
@@ -159,7 +160,6 @@
         'center' )
     ) ,
 '', '#ffffff', 'border="0" cellspacing="0" cellpadding="0" width="100%"' );
-do_hook('login_form');
 echo '</form>' . "\n";
 
 do_hook('login_bottom');
